src/kmeans.jl

```diff
@@ -21,12 +21,31 @@ struct KmeansResult{C<:AbstractMatrix{<:AbstractFloat},D<:Real,WC<:Real} <: Clus
     assignments::Vector{Int}   # assignments (n)
     costs::Vector{D}           # cost of the assignments (n)
     counts::Vector{Int}        # number of points assigned to each cluster (k)
-    cweights::Vector{WC}       # cluster weights (k)
+    wcounts::Vector{WC}        # cluster weights (k)
     totalcost::D               # total cost (i.e. objective)
     iterations::Int            # number of elapsed iterations
     converged::Bool            # whether the procedure converged
 end
 
+# FIXME remove after deprecation period for cweights
+Base.propertynames(clu::KmeansResult, private::Bool = false) =
+    (:centers, :assignments, :costs, :counts, :wcounts,
+     :totalcost, :iterations, :converged,
+     #= deprecated as of 0.13.2 =# :cweights)
+
+# FIXME remove after deprecation period for cweights
+function Base.getproperty(clu::KmeansResult, prop::Symbol)
+    if prop == :cweights
```

Comment:
This might help the compiler a bit:
```suggestion
    if prop === :cweights
```